CHANGELOG.md

```diff
@@ -0,0 +1,7 @@
+# Changelog
+All notable changes to `knotx-fragments` will be documented in this file.
+
+## Unreleased
+List of changes that are finished but not yet released in any final version.
+- [PR-45](https://github.com/Knotx/knotx-fragments/pull/45) - Action log mechanism implementation.
```

Comment:
Nice! Can you please add also `https://github.com/Knotx/knotx-fragments/pull/51` ?

Reply:
If it's not in `master` please add this entry to the `master`, not to this PR.